--- Makefile.orig	Fri Sep  7 09:40:21 2001
+++ Makefile	Wed Oct  9 10:34:31 2002
@@ -7,7 +7,7 @@
 
 
 CFLAGS=-Wall -O2 $(SDL_CFLAGS) -DDATA_PREFIX=\"$(DATA_PREFIX)\" -DDEBUG \
-	-DVERSION=\"$(VERSION)\" -D$(SOUND)SOUND
+	-DVERSION=\"$(VERSION)\" -D$(SOUND)SOUND $(RPM_OPT_FLAGS)
 
 SDL_CFLAGS=$(shell sdl-config --cflags)
 SDL_LIBS=$(shell sdl-config --libs)
@@ -15,8 +15,8 @@
 LIBS=$(SDL_LIBS) $(MIXERLIB) -lSDL_image
 MIXERLIB=-lSDL_mixer
 
-DATA_PREFIX=/usr/share/tuxmath/
-BIN_PREFIX=/usr/local/bin/
+DATA_PREFIX=$(_prefix)/share/tuxmath/
+BIN_PREFIX=$(_prefix)/bin
 
 
 VERSION=2001.09.07
@@ -31,13 +31,9 @@
 install:
 	@echo "COPYING BINARY TO $(BIN_PREFIX)"
 	cp tuxmath $(BIN_PREFIX)
-	chown root.root $(BIN_PREFIX)/tuxmath
-	chmod a+rx,g-w,o-w $(BIN_PREFIX)/tuxmath
 	@echo "COPYING DATA FILES TO $(DATA_PREFIX)"
 	mkdir -p $(DATA_PREFIX)
 	cp -r data/* $(DATA_PREFIX)
-	chown -R root.root $(DATA_PREFIX)
-	chmod -R a+rX,g-w,o-w $(DATA_PREFIX)
 
 clean:
 	-rm tuxmath
